<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网络调试助手 V1.3</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/new/prefix1/images/udp.ico</normaloff>:/new/prefix1/images/udp.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit_MsgList">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>281</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Clinet">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>50</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>客户端</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Server">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>50</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>服务器端</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_IPAddrees">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>90</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentText">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>IP地址:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>监听端口:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_Port">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>130</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="value">
     <number>6657</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>291</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>TCP服务器端与客服端参数选项</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>160</y>
      <width>228</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_StartListen">
         <property name="font">
          <font>
           <family>仿宋</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>启动监听</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/new/prefix1/images/start.ico</normaloff>:/new/prefix1/images/start.ico</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_CloseListen">
         <property name="font">
          <font>
           <family>仿宋</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>关闭监听</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/new/prefix1/images/stop.ico</normaloff>:/new/prefix1/images/stop.ico</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_ConnectServer">
         <property name="font">
          <font>
           <family>仿宋</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>连接服务</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/new/prefix1/images/connect.ico</normaloff>:/new/prefix1/images/connect.ico</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_DisconnectServer">
         <property name="font">
          <font>
           <family>仿宋</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>断开服务</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/new/prefix1/images/disconnect.ico</normaloff>:/new/prefix1/images/disconnect.ico</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>301</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>消息列表</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>631</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>发送测试消息</string>
    </property>
    <widget class="QPlainTextEdit" name="plainTextEdit_SendMsg">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>30</y>
       <width>581</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>380</y>
      <width>312</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton_UdpTest">
       <property name="font">
        <font>
         <family>仿宋</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>UDP调试帮手</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/new/prefix1/images/udp.ico</normaloff>:/new/prefix1/images/udp.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_ClientAutTest">
       <property name="font">
        <font>
         <family>仿宋</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>客服端自动化测试</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>380</y>
      <width>219</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="pushButton_Exit">
       <property name="font">
        <font>
         <family>仿宋</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>退出程序</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/new/prefix1/images/exit.ico</normaloff>:/new/prefix1/images/exit.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_SendMsg">
       <property name="font">
        <font>
         <family>仿宋</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>发送消息</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/new/prefix1/images/sendmsg.ico</normaloff>:/new/prefix1/images/sendmsg.ico</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>plainTextEdit_MsgList</zorder>
   <zorder>radioButton_Clinet</zorder>
   <zorder>radioButton_Server</zorder>
   <zorder>comboBox_IPAddrees</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>spinBox_Port</zorder>
   <zorder>formLayoutWidget</zorder>
   <zorder>horizontalLayoutWidget_3</zorder>
   <zorder>horizontalLayoutWidget_4</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
